validated/dash_apps/finished_apps/daily_validation.py

```diff
@@ -30,10 +32,67 @@
     maximum=maximum,
 )
 
-# Create plot
-x = data["series"]["measurement"]["time"]
-y = data["series"]["measurement"]["average"]
-plot = px.line(x=x, y=y)
+# Detail data
+date: datetime = datetime.strptime("2023-03-14", "%Y-%m-%d")
+data_detail = detail_list(
+    station=station,
+    variable=variable,
+    date_of_interest=date,
+    minimum=minimum,
+    maximum=maximum,
+)
+
+# Tables
+table_daily = create_daily_table(data)
+table_detail = create_detail_table(data_detail)
+
+# Plot
+plot = go.Figure()
+plot.add_trace(
+    go.Scatter(
+        x=data["series"]["measurement"]["time"],
+        y=data["series"]["measurement"]["average"],
+        mode="lines",
+        name="Measurement",
+        line=dict(color="black"),
+    )
+)
+plot.add_trace(
+    go.Scatter(
+        x=data["series"]["selected"]["time"],
+        y=data["series"]["selected"]["average"],
+        mode="lines",
+        name="Selected",
+        line=dict(color="#636EFA"),
+    )
+)
+plot.add_trace(
+    go.Scatter(
+        x=data["series"]["validated"]["time"],
+        y=data["series"]["validated"]["average"],
+        mode="lines",
+        name="Validated",
+        line=dict(color="#00CC96"),
+    )
+)
 
-# Create layout
-app.layout = html.Div([dcc.Graph(figure=plot)])
+# Layout
+app.layout = html.Div(
```

Comment:
I think this results in an odd number of vertical scroll bars.

- I've the external one - the normal one of the browser.
- The internal one to scroll a single table.
- An intermediate one that apparently scrolls through the two tables and the plot. 

I don't think I see the point of this intermediate scrollbar and, moreover, it makes the navigation rather complicated. Is there a way of removing it and leave just two (the browser one and the table specific one)?.
![Screenshot from 2024-02-05 17-27-55](https://github.com/ImperialCollegeLondon/paricia/assets/6095790/90116557-ace0-4bb3-bc58-24059bcc6915)

Reply:
This seems to have fixed itself in a later PR